ts/screens/onboarding/EmailReadScreen.tsx

```diff
@@ -82,62 +91,93 @@ export class EmailReadScreen extends React.PureComponent<Props> {
       .map(_ => untag(_.spid_email))
       .getOrElse("");
 
+    const footerProps1: SingleButton = {
+      type: "SingleButton",
+      leftButton: {
+        bordered: true,
+        onPress: () =>
+          this.props.navigation.navigate(ROUTES.INSERT_EMAIL_SCREEN, {
+            isEditing: true
+          }),
+        title: I18n.t("email.edit.cta")
+      }
+    };
+
+    const footerProps2: TwoButtonsInlineHalf = {
+      type: "TwoButtonsInlineHalf",
+      leftButton: {
+        block: true,
+        bordered: true,
+        title: I18n.t("email.edit.cta"),
+        onPress: this.props.navigateToEmailInsertScreen
+      },
+      rightButton: {
+        block: true,
+        primary: true,
+        title: I18n.t("global.buttons.continue"),
+        onPress: this.props.acknowledgeEmail
+      }
+    };
+
+    const isFromProfileSection = this.props.navigation.getParam(
+      "isFromProfileSection"
+    );
+
     return (
       <TopScreenComponent
-        goBack={this.handleGoBack}
-        title={I18n.t("onboarding.email.read.title")}
+        goBack={
+          isFromProfileSection
+            ? this.props.navigation.goBack
+            : this.handleGoBack
+        }
+        title={I18n.t("profile.preferences.list.email")}
         contextualHelp={{
-          title: I18n.t("onboarding.email.read.title"),
-          body: () => (
-            <Markdown>{I18n.t("onboarding.email.read.help")}</Markdown>
-          )
+          title: I18n.t("email.read.title"),
+          body: () => <Markdown>{I18n.t("email.read.help")}</Markdown>
```

Comment:
Try to move out this lamba to a const function

Reply:
What about a new story to fix it? The same approach is repeated 8 times into the code.

Reply:
I think we should apply a fix whenever we find them.
It is a little contribute to avoid to accumulate technical debt on each PR

Reply:
ok, updated within https://github.com/teamdigitale/io-app/pull/1363/commits/8a6795f071591c8134d10d28f9ff10d0d2ed3b5b

Reply:
I preferred to create a contant to get both contextual help attributes being declared togheter. Any advice on it?

Reply:
I think you approach is right 👍 
```typescript 
{
  title: I18n.t("email.insert.help.title"),
  body: () => <Markdown>{I18n.t("email.insert.help.content")}</Markdown>
}
```